private_repo/ansible/inventory/dev/Core/common.yml

```diff
@@ -20,41 +20,78 @@ sunbird_mail_server_from_email: "support@myorg.com"       # Email ID that should
+cloud_public_storage_endpoint: ""     # Not required in current release
+cloud_public_storage_region: ""       # If cloud_service_provider is aws or oci update this variable
+cloud_public_storage_project: ""      # If cloud_service_provider is gcp update this variable
+cloud_public_storage_namespace: ""    # If cloud_service_provider is oci update this variable
+
+cloud_private_storage_accountname: "" # If cloud_service_provider is aws--> access key, gcp--> service account name, azure--> storage account name
+cloud_private_storage_endpoint: ""    # Not required in current release
+cloud_private_storage_region: ""      # If cloud_service_provider is aws or oci update this variable
+cloud_private_storage_project: ""     # If cloud_service_provider is gcp update this variable
+cloud_private_storage_namespace: ""    # If cloud_service_provider is oci update this variable
+
+cloud_management_storage_accountname: ""  # If cloud_service_provider is aws--> access key, gcp--> service account name, azure--> storage account name
+cloud_management_storage_endpoint: ""     # Not required in current release
+cloud_management_storage_region: ""       # If cloud_service_provider is aws or oci update this variable
+cloud_management_storage_project: ""      # If cloud_service_provider is gcp update this variable
+cloud_management_storage_namespace: ""    # If cloud_service_provider is oci update this variable
+
+cloud_artifact_storage_accountname: ""  # If cloud_service_provider is aws--> access key, gcp--> service account name, azure--> storage account name
+cloud_artifact_storage_endpoint: ""     # Not required in current release
+cloud_artifact_storage_region: ""       # If cloud_service_provider is aws or oci update this variable
+cloud_artifact_storage_project: ""      # If cloud_service_provider is gcp update this variable
+cloud_artifact_storage_namespace: ""    # If cloud_service_provider is oci update this variable
+
+# Create object storage for each below mentioned variables and update accordingly
+cloud_storage_certqr_bucketname: ""
+cloud_storage_chatbot_bucketname: ""
+cloud_storage_dial_bucketname: ""
+cloud_storage_flink_bucketname: ""
+cloud_storage_playercdn_bucketname: ""
+cloud_storage_public_bucketname: ""
+cloud_storage_publicreports_bucketname: ""
+cloud_storage_privatereports_bucketname: ""
+cloud_storage_samiksha_bucketname: ""
+cloud_storage_schema_bucketname: ""
+cloud_storage_sourcing_bucketname: ""
+cloud_storage_offlineinstaller_bucketname: ""
+cloud_storage_content_bucketname: ""
+cloud_storage_telemetry_bucketname: ""
+cloud_storage_termsandcondtions_bucketname: ""
+cloud_storage_user_bucketname: ""
+cloud_storage_desktopappcrashlogs_bucketname: ""
+cloud_storage_label_bucketname: ""
+cloud_storage_certservice_bucketname: ""
+cloud_storage_uci_bucketname: "" 
+cloud_storage_artifacts_bucketname: ""
+
+cloud_storage_management_bucketname: ""  # This object storage is used for backups
```

Comment:
Add a hyperlink to the default section in comment and move all the below commented vars to default.